app/views/api/v2/smart_proxies/main.json.rabl

```diff
@@ -2,11 +2,14 @@ object @smart_proxy
 
 extends "api/v2/smart_proxies/base"
 
-attributes :created_at, :updated_at
-
+attributes :created_at, :updated_at, :ping, :hosts_count
```

Comment:
I see that `ping` entry is `true` in my response, but not sure what does it mean? Does this mean we can actually `ping` the proxy or if it means that all services are up and running and there are no failed features? Probably it's worth to rename the entry to be `status` with value `ok/FAIL` as we do for `/api/statuses` response.

Reply:
The ping is exactly like Communication status in the UI, it means that we can actually can pint the proxy.  I will change the values to OK/ FAIL